test/wallet-account.test.js

```diff
@@ -106,11 +109,26 @@ it('can request transaction signing', async () => {
     });
 });
 
-it('requests transaction signing automatically when there is no local key', async () => {
-    // TODO: Refactor copy-pasted common setup code
-    let keyPair = nearApi.KeyPair.fromRandom('ed25519');
+function parseTransactionsFromUrl(urlToParse) {
+    const parsedUrl = url.parse(urlToParse, true);
+    expect(parsedUrl).toMatchObject({
```

Comment:
I'm never sure if it makes sense to put expectations inside helper functions like this – seems like a side-effect in an otherwise pure function. It's probably fine for now, but I'm curious if you have the same misgivings, @vgrichina

Reply:
Don't see a problem with that. Asserting invariants is common pattern even for non-test code.